libbpf-tools/ksnoop.c

```diff
@@ -251,7 +252,7 @@ static int get_func_btf(struct btf *btf, struct func *func)
 	return 0;
 }
 
-int predicate_to_value(char *predicate, struct value *val)
```

Comment:
I'm fine with this change, but can you confirm that it's independent of the others in this file? If it's not, then I must be confused

Reply:
Yes, it's independent.